main_board/app/tests.conf

```diff
@@ -37,6 +37,8 @@ CONFIG_ZTEST=y
 CONFIG_ZTEST_NEW_API=y
 CONFIG_ZTEST_STACK_SIZE=3000
 
+CONFIG_COVERAGE=y
```

Comment:
from what I can see, `tests.conf` is only used for targets `pearl_main` & `diamond_main` but these don't have COVERAGE support, or am I missing something?

Reply:
done